pkg/services/apiserver/options/extra.go

```diff
@@ -1,9 +1,10 @@
 package options
 
 import (
+	"log/slog"
 	"strconv"
 
-	"github.com/go-logr/logr"
```

Comment:
This still in use or go mod tidy removes it?

Reply:
good catch. it's no longer a direct dependency. updated `go.mod` in [bb15b87](https://github.com/grafana/grafana/pull/84680/commits/bb15b876131e70e607b68eec391aa45497d16be8)